source/extensions/transport_sockets/alts/tsi_socket.h

```diff
@@ -130,7 +130,8 @@ class TsiSocket : public Network::TransportSocket,
 /**
  * An implementation of Network::TransportSocketFactory for TsiSocket
  */
-class TsiSocketFactory : public Network::CommonTransportSocketFactory {
+class TsiSocketFactory : public Network::DownstreamTransportSocketFactory,
+                         public Network::CommonTransportSocketFactory {
```

Comment:
CommonTransportSocketFactory is actually upstream as it only handles transport socket options.

Reply:
Yeap, renaming.